internal/featuretests/v3/global_authorization_test.go

```diff
@@ -14,6 +14,7 @@
 package v3
 
 import (
+	"k8s.io/apimachinery/pkg/types"
```

Comment:
Please run `make format` to move this import to the expected section and fix the linter error.